manifests/k/Kubecolor/kubecolor/0.4.0/Kubecolor.kubecolor.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Kubecolor.kubecolor
+PackageVersion: 0.4.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: kubecolor.exe
+  PortableCommandAlias: kubectl
+Installers:

```

Comment:
```suggestion
  PortableCommandAlias: kubecolor
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Kubernetes.kubectl
Installers:
```

Reply:
Oh I didnt know you could do dependencies, that is great and yes makes total sense to add it there.

> I'm concerned that using kubectl for the symlink might be confusing

Since it creates symlink, yes it would be an isssue. It essentially needs to be alias for kubectl, which whoever needs it will add to their powershell profile.